testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -4174,6 +4174,37 @@ static void checkRlikeFails(SqlOperatorFixture f) {
     f0.forEachLibrary(libraries, consumer);
   }
 
+  @Test void testToHex() {
+    final SqlOperatorFixture f0 = fixture().setFor(SqlLibraryOperators.TO_HEX);
+    f0.checkFails("^to_hex(x'')^",
+        "No match found for function signature TO_HEX\\(<BINARY>\\)",
+        false);
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.BIG_QUERY);
+    f.checkString("to_hex(x'00010203AAEEEFFF')",
+        "00010203aaeeefff",
+        "VARCHAR NOT NULL");
+    f.checkString("to_hex(x'')", "", "VARCHAR NOT NULL");
+    f.checkNull("to_hex(cast (null as varbinary))");
```

Comment:
I noticed that there is a blank space after `cast`, shall we remove it? So as the following test.